### ISSUE TITLE
Use OPEM to model water electrolysis

### ISSUE THREAD

**Issue by DanWBR** [ECSIM/opem#172]
Hello,

Is it possible to use OPEM to model water electrolysis, that is, the opposite of a fuel cell?

**Comment by sarminh** [comment]
Dear Daniel, 
The electrolyzer is an apparatus that produces hydrogen (H2) through a chemical process (electrolysis) capable of separating the hydrogen and oxygen molecules of which water is composed using electricity, but in OPEM software we provide useful mathematical models for simulation of fuel cell efficiency to produce electricity from H2 gas fuel. Of course, if we used different functions, we could simulate hydrogen production efficiency in an electrolyzer device. In conclusion, yes it's possible to use OPEM for electrolyzer simulation too, but by consideration of practical mathematical functions. 
Regards.

**Comment by sarminh** [comment]
> Hello,
> 
> Is it possible to use OPEM to model water electrolysis, that is, the opposite of a fuel cell?

Dear Daniel,
The electrolyzer is an apparatus that produces hydrogen (H2) through a chemical process (electrolysis) capable of separating the hydrogen and oxygen molecules of which water is composed using electricity, but in OPEM software we provide useful mathematical models for simulation of fuel cell efficiency to produce electricity from H2 gas fuel. Of course, if we used different functions, we could simulate hydrogen production efficiency in an electrolyzer device.
In conclusion, yes it's possible to use OPEM for electrolyzer simulation too, but by consideration of practical mathematical functions. In fact, if you need to simulate electrolyzer efficiency and H2 production, you should use other practical functions and change the basic construction of the code.
Regards.

**Comment by DanWBR** [comment]
@sarminh thank you very much. I'm currently evaluating OPEM to include PEMFCs in my open-source chemical process simulator DWSIM (https://github.com/DanWBR/dwsim) as clean energy unit operations.

**Comment by sarminh** [comment]
> @sarminh thank you very much. I'm currently evaluating OPEM to include PEMFCs in my open-source chemical process simulator DWSIM (https://github.com/DanWBR/dwsim) as clean energy unit operations.

Very good, collaborating together makes us happy. Please feel free to contact us.

**Comment by DanWBR** [comment]
@sarminh I will, thanks 👍🏻
